local/tests/run_afl_tests

```diff
@@ -8,4 +8,4 @@ if [ "$(cat /proc/sys/kernel/core_pattern)" != "core" ]; then
 fi
 
 AFL_INTEGRATION_TESTS=1 python butler.py py_unittest \
-    -t core -p '*afl_launcher_integration*' "$@"
+    -t core -p '*afl*' "$@"
```

Comment:
wont this run all afl tests not just the integration tests?
Dont the AFL unittests get run elsewhere?